py/desitarget/sv3/sv3_cuts.py

```diff
@@ -1237,6 +1232,21 @@ def isBGS_colors(rfiberflux=None, gflux=None, rflux=None, zflux=None, w1flux=Non
         primary = np.ones_like(rflux, dtype='?')
     bgs = primary.copy()
     fmc = np.zeros_like(rflux, dtype='?')
+    
+    # the LSLGA galaxies.
```

Comment:
While we're updating `isBGS_lslga()`->`isBGS_sga()`, this comment should be updated to "SGA" instead of "LSLGA".